pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -115,8 +115,8 @@ int kprobe__tcp_sendmsg(struct pt_regs* ctx) {
     return 0;
 }
 
-SEC("kretprobe/tcp_sendmsg")
-int kretprobe__tcp_sendmsg(struct pt_regs* ctx) {
+SEC("kretprobe/sk_filter_trim_cap")
+int kretprobe__sk_filter_trim_cap(struct pt_regs* ctx) {
```

Comment:
shall we switch to tcp_filter ?

Reply:
I tried `tcp_filter` yesterday, but it doesn't get executed on my VM, so I'm guessing it's probably inlined in my Kernel? As you mentioned privately it is an exported symbol, so maybe I need to dig further to figure out what's going on.

And it turns out `sk_filter_trim_cap` is a PMU-kprobe type, and which is not supported on older Kernels, so I'll try to find an alternative.